client/src/lib/createClassMap.js

```diff
@@ -0,0 +1,11 @@
+const createClassMap = (classesStr) => {
+  const classConfig = {};
+  if (classesStr) {
+    classesStr.split(' ').forEach(c => {
+      classConfig[c.trim()] = true;
```

Comment:
`c` to `classStr`
don't need `trim()` because the `split(' ')` would do the trimming for you, if wanted could add a `classStr !== ''` check